### DISCUSSION TITLE
Backpack Pro in production

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#990] [Q&A (Help)] [answered]
I need clarification about deploying a project that uses Backpack Pro in production.
The instructions here https://backpackforlaravel.com/user/tokens seem to indicate installing in development environment but in production?
Do I need to put a key on the .env file?
Perhaps it would be better to add some specific information in your documentation

**Comment by karandatwani92** [comment, marked as answer]
Hey @blondie63

On the local machine, you installed Backpack packages using `composer install` & `composer update`.
You'll be doing the same on your online/production machine.

So, access to the download package is feasible only with the credentials. You need to put the credentials in **auth.json** to download the package to that online machine.

Just follow the same [instructions](https://backpackforlaravel.com/user/tokens) you did on the local machine.

We needed `BACKPACK_LICENSE` in .ENV only on older Backpack v4.

Feel free to ask any other queries.
